RELEASE-NOTES.txt

```diff
@@ -347,4 +347,4 @@ o COLLECTIONS-217:  Calling "setValue(Object)" on any Entry returned by a "Flat3
 For complete information on Apache Commons Collections, including instructions on how to submit bug reports,
 patches, or suggestions for improvement, see the Apache Commons Collections website:
 
-https://commons.apache.org/collections/
+https://commons.apache.org/proper/commons-collections/
```

Comment:
:+1: less one hop to reach the page.

Note, however, that this file is automatically re-generated by the commons-release-plugin: https://github.com/apache/commons-release-plugin/blob/3a0a64db33251d5a3bc808790d140f7db7347157/src/changes/release-notes.vm

A release-manager follows steps defined in this page, which includes other commands that generate this and other files: https://commons.apache.org/releases/prepare.html

Reply:
File has been rollbacked.